src/SonarScanner.MSBuild.PreProcessor/SonarWebService.cs

```diff
@@ -31,7 +31,7 @@
 
 namespace SonarScanner.MSBuild.PreProcessor
 {
-    public sealed class SonarWebService : ISonarQubeServer
+    public sealed class SonarWebService : ISonarWebService
```

Comment:
This line is the main motivation for this PR - deconfusion by aligning the names